include/ftl/task_scheduler.h

```diff
@@ -63,13 +62,11 @@ class TaskScheduler {
 	~TaskScheduler();
 
 private:
-	enum : size_t {
```

Comment:
I would prefer the old style. If it means we have to force all one liners to be expanded, that would be my choice

Reply:
I currently have all one liners that clang-format will expand set to expand. This particular example I replace with a constexpr static variable anyway. I solve this problem with enums by putting an empty comment after the brace to force it into using the "one per line" mode, as it won't break up comments.